app/presenters/tree_builder_automate.rb

```diff
@@ -3,8 +3,8 @@ def tree_init_options
     {:full_ids => false}
   end
 
-  def initialize(name, type, sandbox, build = true, controller = nil)
-    @controller = controller
+  def initialize(name, type, sandbox, build = true, **params)
```

Comment:
Is this ever used?

Not seeing any TreeBuilderAutomate calls with `:controller`